lib/utils/error-handler.js

```diff
@@ -106,12 +106,21 @@ function writeLogFile(cb) {
     , sys = require("util");
 
   log.history.forEach(function(m) {
-    var lvl = log.LEVEL[m.level]
-      , pref = m.pref ? " " + m.pref : ""
-      , b = lvl + pref + " "
-      , msg = typeof m.msg === "string" ? m.msg
-        : msg instanceof Error ? msg.stack || msg.message
-        : sys.inspect(m.msg, 0, 4);
+    var msg;
+    if (typeof m.msg === "string") {
+      msg =  m.msg;
+    } else {
+      if (msg instanceof Error) {
```

Comment:
I think a level of nesting can be removed here by bringing this `if` up to the `else` above it to make an `else if`. Does that sound right?